packages/d3fc-series/src/webgl/area.js

```diff
@@ -0,0 +1,50 @@
+import xyBase from '../xyBase';
+import { glArea, scaleMapper } from '@d3fc/d3fc-webgl';
+import { rebindAll, exclude, rebind } from '@d3fc/d3fc-rebind';
+
+export default () => {
+    const base = xyBase();
+
+    let draw = glArea();
```

Comment:
`const`

Reply:
The other files in `src/webgl` all use `let` here, apart from `bar.js`. Shall I leave this for now to be consistent with the majority of the code base and it can be looked at in the mop up?